protocol/v2/ssv/runner/attester.go

```diff
@@ -166,10 +160,15 @@ func (r *AttesterRunner) ProcessPostConsensus(logger *zap.Logger, signedMsg *spe
 		}
 		specSig := phase0.BLSSignature{}
 		copy(specSig[:], sig)
-
+		r.metrics.EndPostConsensus()
 		logger.Debug("🧩 reconstructed partial signatures",
-			zap.Uint64s("signers", getPostConsensusSigners(r.GetState(), root)))
+			zap.Uint64s("signers", getPostConsensusSigners(r.GetState(), root)),
+			fields.QuorumTime(r.metrics.GetPostConsensusTime()))
```

Comment:
don't you think its confusing to use QuorumTime with same key for pre, consensus and post? just for sake of easy search or comparing results, or making averages, it makes sense to add some prefix like pre_/cons_/post_ to it.

Reply:
done